openassessment/tests/test_data.py

```diff
@@ -152,3 +225,124 @@ def _load_fixture(self, fixture_relpath):
         )
         print "Loading database fixtures from {}".format(fixture_path)
         call_command('loaddata', fixture_path)
+
+
+class TestOraAggregateData(TransactionCacheResetTest):
+    """
+    Test that the component parts of OraAggregateData behave as expected.
+    """
```

Comment:
Maybe mention that this is an integration test, testing against the real submissions api.